doc/lua_api.md

```diff
@@ -3928,6 +3928,28 @@ The following functions provide escape sequences:
     * Removes all color escape sequences.
 
 
+Coordinate System
+=================
+
+Luanti uses a **left-handed** coordinate system: Y is "up", X is "right", Z is "forward".
+This is the convention used by Unity, DirectX and Irrlicht.
+It means that when you're pointing in +Z direction in-game ("forward"), +X is to your right; +Y is up.
+
+Consistently, rotation is **left-handed** as well:
+When your thumb points in the direction of the rotation axis,
+your curled fingers point in the direction the rotation goes.
+The rotation order is XYZ: First rotation around the X-axis is applied, then Y, then Z.
```

Comment:
This alone is confusing because it's not clear if it's the unrotated or the rotated axis.

```suggestion
The rotation order is extrinsic XYZ: First rotation around the (unrotated) X-axis is applied, then Y, then Z.
```